From 7115ca05b23d66ce5a88e52a05b7813443f5059e Mon Sep 17 00:00:00 2001
From: Stephen Hemminger <stephen@networkplumber.org>
Date: Tue, 9 Aug 2022 13:27:33 -0700
Subject: [PATCH 2/2] ipstats: add missing headers

IWYU reports several headers are not explicitly
included by ipstats.

Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
---
 ip/ipstats.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ip/ipstats.c b/ip/ipstats.c
index 1ac275bd..dadded14 100644
--- a/ip/ipstats.c
+++ b/ip/ipstats.c
@@ -1,7 +1,11 @@
 // SPDX-License-Identifier: GPL-2.0+
+#include <alloca.h>
 #include <assert.h>
 #include <errno.h>
+#include <stdio.h>
+#include <string.h>
 #include <sys/param.h>
+#include <sys/socket.h>
 
 #include "list.h"
 #include "utils.h"
-- 
2.37.3

